src/components/Header.js

```diff
@@ -0,0 +1,11 @@
+import React from 'react';
+
+const Header = () => {
```

Comment:
I like this new header component 👏  
I'm starting to think what to say there for when we merge (maybe the app name?) 🤔

Reply:
Yes! We need to come up with a cool name for the app. 😃